simple_pdf_generator/src/lib.rs

```diff
@@ -147,23 +144,14 @@ static BROWSER: Lazy<RwLock<Option<ChromiumInstance>>> = Lazy::new(|| RwLock::ne
 static TOKENS_AND_IMAGES_REGEX: Lazy<Regex> = Lazy::new(|| {
     Regex::new(r#"(?:%%(?P<prop_name>.*)%%)|(?:<img[^>]*\ssrc="(?P<img_src>.*?)"[^>]*>)"#).unwrap()
 });
-static NO_SANDBOX: AtomicBool = AtomicBool::new(false);
 
-pub fn set_no_sandbox(val: bool) {
-    NO_SANDBOX.store(val, Ordering::Relaxed);
-}
 
-pub async fn generate_pdf(
+pub async fn generate_pdf_from_html(
```

Comment:
A `&str` is more correct because the parameter isn't mutable. If you prefer you can leave it as is and I will change it before the release

Reply:
I would appreciate it if you could